### ISSUE TITLE
urlib3 version incompatibility

### ISSUE THREAD

**Issue by voisey** [magma/magma#13620]
After spinning up the magma-dev VM on master and running `cd $MAGMA_ROOT/lte/gateway && make run`, I get the following errors:
```
Installing egg link for lte
/home/vagrant/build/python/bin/pip3 install -q -U --cache-dir ~/.pipcache --no-build-isolation -e .[dev]
ERROR: sentry-sdk 1.9.4 has requirement urllib3>=1.26.11; python_version >= "3.6", but you'll have urllib3 1.25.8 which is incompatible.
ERROR: sentry-sdk 1.9.4 has requirement urllib3>=1.26.9; python_version >= "3.5", but you'll have urllib3 1.25.8 which is incompatible.
make[2]: Leaving directory '/home/vagrant/magma/lte/gateway/python'
```
Note that the script continues to run nevertheless.

**Comment by voisey** [comment]
Looks like this will be fixed by https://github.com/magma/magma/pull/13482.

**Comment by mpfirrmann** [comment]
Closed with #13482.
